walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
```

Comment:
The changes to this and the script.js file seem unrelated and should probably not be in this PR, though maybe github is confusing me again somehow.

Reply:
That was a change I made during last week's project. I am confused as well as why that is shown as a change in this PR.

Reply:
Could you revert these changes if easily possible? If not, it's fine. I _think_ it should be as simple as doing a new commit that changes the names back.

Reply:
I'm a little confused - the java code you're adding in this CL is unrelated to the HTML/JS code, and furthermore it doesn't look like anything in the HTML/JS is even fetching from /data.